docs/nlu/components.rst

```diff
@@ -922,7 +922,9 @@ EmbeddingIntentClassifier
          | drop_rate                       | 0.2              | Dropout rate for encoder. Value should be between 0 and 1.   |
          |                                 |                  | The higher the value the higher the regularization effect.   |
          +---------------------------------+------------------+--------------------------------------------------------------+
-         | use_sparse_input_dropout        | True             | If 'True' apply dropout to sparse tensors.                   |
+         | use_sparse_input_dropout        | True             | If 'True' apply dropout to sparse input tensors.             |
+         +---------------------------------+------------------+--------------------------------------------------------------+
+         | use_dense_input_dropout         | True             | If 'True' apply dropout to dense input tensors.              |
```

Comment:
Isn't the default value False here? Even `use_sparse_input_dropout` is False

Reply:
sparse is true here, so I put dense to true as well

Reply:
The defaults in EmbeddingIntentClassifier have False for both

Reply:
oh, that's a mistake, fixed it